packages/preview/modern-sustech-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "modern-sustech-thesis"
+version = "0.1.0"
+entrypoint = "template/main.typ"
```

Comment:
This is not really how the template mechanism was intended. Generally, all of the formatting code should live in the package itself and not in the `template` directory and the `template` directory contains just the files that will be cloned into the user project. There you then have a `show:` rule that applies your template.

You can take a look at the `charged-ieee` template or some of the existing thesis templates to see how it is supposed to look like.

Reply:
Thanks, I will construct my template a new version and upload it soon.

Reply:
I finished the new version upload, please review it at your convenience!